### PR TITLE
Fix downcase, add safe area

### PR DESCRIPTION
**Pull request by igorkislyuk**
- Add cell template naming `cell-template`
- Fix issue with naming. Now methods for controller named `createXXXXViewController()`
- Add safe area

// example of Rambafile

```
### Headers settings
company: Touch Instinct

### Xcode project settings
project_name: JetExchange
xcodeproj_path: JetExchange.xcodeproj

### Code generation settings section
# The main project target name
project_target: JetExchange

# The file path for new modules
project_file_path: JetExchange/Controllers

# The Xcode group path to new modules
project_group_path: JetExchange/Controllers

### Dependencies settings section
podfile_path: Podfile

catalogs:
- 'https://github.com/igorkislyuk/generamba-templates'

### Templates

templates:
- {name: touchin}
- {name: touchin-tablecell}
```

### CODE REVIEW

File: touchin-cell/Cell/cell.swift.liquid [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import UIKit
+import TableKit
+import RxSwift
+import RxCocoa
+
+/// <#Cell description#>
+final class {{ module_info.name }}: UITableViewCell, ConfigurableCell {
+
+    fileprivate var disposeBag = DisposeBag()
```
Comment (petropavel13) [inline touchin-cell/Cell/cell.swift.liquid]: in Swift 4 you can use just `private`

File: touchin-cell/Cell/cellViewModel.swift.liquid [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import RxSwift
+import RxCocoa
+
+final class {{ module_info.name }}ViewModel {
```
Comment (petropavel13) [inline touchin-cell/Cell/cellViewModel.swift.liquid]: final class without inheritance almost the same as struct. use struct by default?

File: touchin-cell/Cell/view.xib.liquid [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13173"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <tableViewCell contentMode="scaleToFill" selectionStyle="none" indentationWidth="10" rowHeight="44" id="KGk-i7-Jjw" customClass="{{ module_info.name }}">
```
Comment (petropavel13) [inline touchin-cell/Cell/view.xib.liquid]: touchin-cell/Cell/view.xib.liquid

maybe

touchin-tablecell/TableCell/view.xib.liquid

**Review by petropavel13 — CHANGES_REQUESTED** [review]
You can also add some `closes #X` words to PR or commits to make a links

File: touchin-cell/Cell/cell.swift.liquid [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        clear()
```
Comment (scoreyou) [inline touchin-cell/Cell/cell.swift.liquid]: I agree with guys, think clear() in awakeFromNib is redundant

**Review by scoreyou — CHANGES_REQUESTED** [review]

File: touchin-cell/Cell/cell.swift.liquid (hunk 1 above)
Comment (igorkislyuk) [reply touchin-cell/Cell/cell.swift.liquid]: Not all projects in swift 4.

File: touchin-cell/Cell/view.xib.liquid (hunk 3 above)
Comment (igorkislyuk) [reply touchin-cell/Cell/view.xib.liquid]: Sounds good. ty

File: touchin-cell/Cell/cellViewModel.swift.liquid (hunk 2 above)
Comment (igorkislyuk) [reply touchin-cell/Cell/cellViewModel.swift.liquid]: No. Struct got different purpose.

**Review by iON1k — APPROVED** [review]

**Review by scoreyou — APPROVED** [review]

**Review by petropavel13 — APPROVED** [review]

**Pull request merged**